index.js

```diff
@@ -173,7 +173,24 @@ const create = (win, options) => {
 
 		const shouldShowInspectElement = typeof options.showInspectElement === 'boolean' ? options.showInspectElement : isDev;
 
+		const dictSuggestions = [];
+		for (let index = 0; index < props.dictionarySuggestions.length; index++) {
```

Comment:
`Array#map` would be more appropriate here.

Reply:
Performance Analysis For loops performs faster than map or foreach as number of elements in a array increases. https://stackoverflow.com/a/56275105

Reply:
That is irrelevant for such a small array. You would have to run through tens of thousands of elements for that to matter.